packages/config/src/tokens/script.ts

```diff
@@ -0,0 +1,252 @@
+import { getEnv } from '@l2beat/backend-tools'
+import { CoingeckoClient, HttpClient } from '@l2beat/shared'
+import { AssetId, ChainId, Token } from '@l2beat/shared-pure'
+import { providers } from 'ethers'
+import { readFileSync, writeFileSync } from 'fs'
+import { parse, ParseError } from 'jsonc-parser'
+
+import { chains } from '../chains'
+import { ChainConfig } from '../common'
+import { Output, Source, SourceEntry } from './types'
+import { getCoingeckoId } from './utils/getCoingeckoId'
+import { getTokenInfo } from './utils/getTokenInfo'
+import { ScriptLogger } from './utils/ScriptLogger'
+
+const SOURCE_FILE_PATH = './src/tokens/tokens.jsonc'
+const OUTPUT_FILE_PATH = './src/tokens/generated.json'
+
+async function main() {
+  const logger = new ScriptLogger({})
+  const coingeckoClient = getCoingeckoClient()
+  const source = getSourceFile(logger)
+  const output = getOutputFile(logger)
+  const result: Token[] = []
+
+  for (const [devId, tokens] of Object.entries(source)) {
+    const chainLogger = logger.prefix(devId)
+    const chain = getChainConfiguration(chainLogger, devId)
+    const chainId = getChainId(chainLogger, chain)
+
+    chainLogger.processing()
+
+    for (const token of tokens) {
+      const tokenLogger: ScriptLogger = chainLogger.addMetadata(token.symbol)
+
+      const type = getType(tokenLogger, devId, token)
+      const formula = getFormula(tokenLogger, devId, token)
+      const category = token.category ?? 'other'
+
+      const existingToken = findTokenInOutput(output, chainId, token)
+
+      if (existingToken) {
+        tokenLogger.skipping()
+        result.push({
```

Comment:
Add commend about config values overriding output

Reply:
Maybe coingeckoId here is missing

Reply:
c72dc72c12df62ee678aa14dcb0340efa71bb659